### PR TITLE
Test scripts

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add tests for the Dataset_To_plate and Kymograph_analysis

Also rename the test file for figure export scripts

Add test for Populate metadata

# Testing this PR

Check that the tests are green

### CODE REVIEW

File: components/tools/OmeroPy/test/integration/scriptstest/test_util_scripts.py [hunk 1]
```diff
@@ -112,3 +113,30 @@ def test_images_from_rois(self):
...
+
+        # create several test images in a dataset
+        dataset = self.make_dataset("dataset_to_plate-test", client=client)
+
+        for i in range(10):
+            # x,y,z,c,t
+            image = self.create_test_image(100, 100, 1, 1, 1, session)
+            self.link(dataset, image, client=client)
+
+        # run the script twice. First with all args...
```
Comment (mtbc) [inline components/tools/OmeroPy/test/integration/scriptstest/test_util_scripts.py:134]: did I miss the second time?

**Comment by jburel** [comment]
https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-python/ is green

File: components/tools/OmeroPy/test/integration/scriptstest/test_util_scripts.py [hunk 2]
```diff
@@ -112,3 +113,30 @@ def test_images_from_rois(self):
...
+        dataset_ids = [omero.rtypes.rlong(dataset.getId().getValue())]
+        args = {
+            "Data_Type": omero.rtypes.rstring("Dataset"),
+            "IDs": omero.rtypes.rlist(dataset_ids)
+        }
+
+        d_to_p = run_script(client, script_id, args, "New_Object")
+        # check the result
+        assert d_to_p is not None
+        assert d_to_p.getValue().getId().getValue() > 0
```
Comment (will-moore) [inline components/tools/OmeroPy/test/integration/scriptstest/test_util_scripts.py]: Maybe nice to check that a Plate got created and it has some Wells & Images?
E.g. 
```
qs = client.getSession().getQueryService()
query = "select well from Well as well left outer join fetch well.plate as pt left outer join fetch well.wellSamples as ws left outer join fetch ws.image as img where well.plate.id = :oid"
params = omero.sys.ParametersI()
params.addLong('oid', omero.rtypes.rlong(plateId))
wells = qs.findAllByQuery(query, params)
assert len(wells) > 0       # or however many we're expecting
# Some check that Wells have Images, E.g.
for w in wells:
    print w.getWellSample(0).getImage().id.val
```

**Comment by will-moore** [comment]
All tests passing today. E.g. [scriptstest.test_analysis_scripts](https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-breaking-integration-Python27/104/testReport/OmeroPy.test.integration.scriptstest.test_analysis_scripts/).

File: components/tools/OmeroPy/test/integration/scriptstest/test_util_scripts.py (hunk 2 above)
Comment (jburel) [reply components/tools/OmeroPy/test/integration/scriptstest/test_util_scripts.py]: @will-moore: good idea

**Comment by jburel** [comment]
https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-python/434/ is green

**Comment by bramalingam** [comment]
The current test looks good. I will add to this test when I reopen my previous PR, https://github.com/ome/scripts/pull/117. This is a specific case where the initial fake_image does not have the time_increment detail. So in its current state, with the tests passing, this PR looks good to merge.

**Comment by jburel** [comment]
@will-moore: are you happy with the adjusted test?

**Comment by will-moore** [comment]
Looks good, yes, Thanks.

**Pull request merged**